### DISCUSSION TITLE
Shadowing the repo and making changes in the repo

### DISCUSSION THREAD

**Discussion by urstrulyvishtan** [LekoArts/gatsby-themes#783] [Questions]
Hello,
So this template contains that last copyright and template by lekoarts in the footer section, I tried to override it and it worked in localhost but I couldn't deploy to vercel have to clear the whole cache and package.json and install npm again, which made that footer section come again.
I searched all over the internet but couldn't find this solution and found a discussion regarding this which didn't help much 
please can someone reply to this thread

**Comment by ch4nd4n** [comment]
What's the question if you have already resolved it?
